example/Dll.c

```diff
@@ -166,6 +146,49 @@ void onFrame(AIModule* self) {
     Iterator_release(units);
 }
 
+void getEventsTest() {
+    // getEvents test
+    Iterator* const events = (Iterator*) Game_getEvents(Broodwar);
+    assert(events);
+
+    for(; Iterator_valid(events); Iterator_next(events)) {
```

Comment:
print only the first event because in OpenBW this code really spams in console. In Starcraft you can see it, because the game shows only last 3-5 messages from you.